src/pages/commitqueue/codeChangesModule/CodeChangeItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { FileDiff } from "types/patch";
+import { StyledLink } from "components/styles/StyledLink";
+import { uiColors } from "@leafygreen-ui/palette";
+
+const { green, red, gray } = uiColors;
+const CodeChangeItemRow = styled("div")`
+  display: grid;
+  ${(props: { isLastItem: boolean }) =>
+    !props.isLastItem && `border-bottom: 1px solid ${gray.light2};`}
+  grid-template-columns: 4fr repeat(2, 1fr);
+  grid-template-rows: 1fr;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  width: 70%;
+`;
+
+const CodeChangeItemGridField = styled("div")`
+  grid-area: ${(props: { grid: string }) => props.grid};
+  margin: 16px 0 16px 0;
+`;
+
+const FileDiffTextContainer = styled("span")`
+  ${(props: { type: string; hasValue: boolean }) =>
+    props.hasValue &&
+    (props.type === "+" ? `color: ${green.base};` : `color: ${red.base};`)}
+  &:nth-child(2) {
+    margin-left: 16px;
+  }
+`;
+interface FileDiffTextProps {
+  type: string;
+  value: number;
+}
+export const FileDiffText: React.FC<FileDiffTextProps> = ({ value, type }) => {
```

Comment:
Instead of creating your own component, did you look at generalizing the CodeChanges component? That one needs a mode where it doesn't display column headers, but otherwise these 2 components have the same data source and are functionally identical

Reply:
all typography should also come from leafygreen https://mongodb.github.io/leafygreen-ui/?path=/story/typography--default